src/inference/tests/functional/caching_test.cpp

```diff
@@ -126,6 +126,8 @@ class MockExecutableNetwork : public IExecutableNetworkInternal {
     MOCK_METHOD0(CreateInferRequest, IInferRequestInternal::Ptr());
     MOCK_CONST_METHOD0(GetInputsInfo, ConstInputsDataMap());
     MOCK_CONST_METHOD0(GetOutputsInfo, ConstOutputsDataMap());
+    MOCK_CONST_METHOD0(getInputs, const std::vector<std::shared_ptr<const ov::Node>>&());
```

Comment:
[ilyachur](https://github.com/ilyachur) [Jul 7, 2023](https://github.com/openvinotoolkit/openvino/pull/18392#discussion_r1255248077)
```
Why do you need this change?
```

Member Author
@[vurusovs](https://github.com/vurusovs) vurusovs [Jul 10, 2023](https://github.com/openvinotoolkit/openvino/pull/18392#discussion_r1258144411)

```
Since Hetero CompiledModel restores inputs/outputs from compiled submodels, I need inputs/outputs information from submodel (it is done according to GetInputsInfo/GetOutputsInfo)
```